algorithms/src/polycommit/kzg10/mod.rs

```diff
@@ -132,7 +130,8 @@ impl<E: PairingEngine> KZG10<E> {
 
         let mut randomness = KZGRandomness::empty();
         if let Some(hiding_degree) = hiding_bound {
-            let mut rng = rng.ok_or(PCError::MissingRng)?;
+            let mut rng =
+                rng.ok_or(anyhow::anyhow!("The provided polynomial was meant to be hiding, but `rng` was `None`"))?;
```

Comment:
Nitpick: polynomials aren't hiding or otherwise, commitments are. Perhaps:

```suggestion
                rng.ok_or(anyhow::anyhow!("The polynomial commitment was meant to be hiding, but `rng` was `None`"))?;
```